src/fourcipp/fourc_input.py

```diff
@@ -63,19 +65,22 @@ def is_section_known(section_name):
 class FourCInput:
     """4C inout file object."""
 
-    known_sections = ALL_SECTIONS
-    type_converter = CONVERTER
+    known_sections: list = ALL_SECTIONS
+    type_converter: Converter = CONVERTER
 
     def convert_to_native_types(self):
         """Convert all sections to native Python types."""
         self._sections = self.type_converter(self._sections)
         self._legacy_sections = self.type_converter(self._legacy_sections)
 
-    def __init__(self, sections=None):
+    def __init__(
+        self,
+        sections: Optional[dict] = None,
+    ):
```

Comment:
```suggestion
    ) -> None:
```

I think if we add type hints we should be consistent and also always add the return type None?

Reply:
Personally, it seems strange, but I guess [this](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions) makes sense. If nothing is indicated means anything can be returned.

Reply:
Yes, everywhere where nothing is returned we have to add this